src/integrationTest/groovy/wooga/gradle/unity/UnityPluginIntegrationSpec.groovy

```diff
@@ -339,4 +339,13 @@ class UnityPluginIntegrationSpec extends UnityIntegrationSpec {
         result.wasSkipped("returnUnityLicense")
         result.wasExecuted("activateUnity")
     }
+
+    @UnityPluginTestOptions(forceMockTaskRun = false, disableAutoActivateAndLicense = false)
+    def "runs generateSolution task from cli"() {
```

Comment:
I don't the task is necessarily invoked from the CLI. We could use a GUI to have them executed!

Reply:
Yeah, in this case, you are right.

Reply:
Related to the other conversation:

In this case, we are testing that the `generateSolution` task is generated by the plugin.